kolibri/plugins/coach/assets/test/useFetchTree.fixtures.js

```diff
@@ -0,0 +1,4708 @@
+export const fetchTreeTopicResponseWithMore = {
```

Comment:
Could you add a code comment here to describe how you generated this file in case it needs to be updated in future?